dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -33,7 +33,8 @@ RUN apt-get update && apt-get install -y build-essential \
                                          telnet \
                                          vim
 
-RUN pip install cffi==1.10.0 \
+RUN pip install allure-pytest==2.8.22 \
+		cffi==1.10.0 \
```

Comment:
Please align this line with the others (it appears there are tabs here when there should be spaces).

Reply:
fixed